synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -81,29 +126,38 @@ def lint_setup_py(session):
     session.run("python", "setup.py", "check", "--restructuredtext", "--strict")
 
 
+def install_unittest_dependencies(session, *constraints):
+    session.install(*UNIT_TEST_STANDARD_DEPENDENCIES, *constraints)
```

Comment:
I'd prefer if we installed UNIT_TEST_STANDARD_DEPENDENCIES and UNIT_TEST_DEPENDENCIES all in one go.

Reply:
Hmm, I'm not sure I see why?  The prior template alwasy installed the "standard" deps (`"mock", "asyncmock", "pytest", "pytest-cov", "pytest-asyncio"`) in a separate step, prior to any others.

Reply:
The reason is that the pip resolver can run in one go. Otherwise subsequent pip installs might have to downgrade some packages.